src/pages/NotificationsPage/components/notification-dropdown.tsx

```
@@ -0,0 +1,285 @@
+                    key={notification.id}
+                    {...notification}
+                    onAction={() => console.log("Action clicked")}
+                  />
+                ))}
+              </div>
+            </>
+          ) : (
+            /* Empty State */
+            <div className="flex flex-col items-center justify-center px-4 py-12">
+              <div className="mb-4 flex h-32 w-32 items-center justify-center">
+                <svg viewBox="0 0 200 200" className="h-full w-full">
```

Biome (2.3.13)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)